### ISSUE TITLE
Widgets missing

### ISSUE THREAD

**Issue by milkmiruku** [forrestguice/SuntimesWidget#258]
Around a month ago, a Suntimes upgrade via Fdroid failed half way through due to the being no more space on my phone. I sorted that, but after reinstalling the app, I could no longer see the associated widgets. I tried uninstalling and installing from Google Play, but still no widgets. Kinda confusing. Might some lingering old files cause this? Or maybe something else?

**Comment by forrestguice** [comment]
That is very unusual. I'm not really sure what to make of it.

You installed from Google Play? That has me confused. I do have a Google Play dev account, but I don't distribute there, and the Suntimes app has always been an F-Droid exclusive.

I'm curious what version is printed on the About dialog? The latest is v0.9.3.

**Comment by milkmiruku** [comment]
Ah, I did not reinstall from Google Play! Whoops, I confused myself and conflated a different recent memory of reinstalling an app lost after an F-Droid upgrade failure.

I've just upgraded to 0.9.3 via F-Droid, the app works fine via the icon, but still no widgets.

I forgot to mention that I'm using Nova Launcher, though I had a widget on my homescreen for several months before the upgrade failure. The rom is Resurrection Remix OS, Android version 4.4.3.

**Comment by milkmiruku** [comment]
Bingo! A quick move of the app back to internal storage and I see the widgets in the available widgets page.

I think on a previous phone I was able to set the installation location by `adb`, but I think that became restricted in later versions, at least I couldn't change the default the same way that last time I tried. Maybe another aspect of the alternative ROM is an ability to move some apps to external, though there are still some apps though that do not give me that option.
